Lib/_pydatetime.py

```diff
@@ -1809,7 +1809,9 @@ def utcfromtimestamp(cls, t):
         warnings.warn("datetime.datetime.utcfromtimestamp() is deprecated and scheduled "
                       "for removal in a future version. Use timezone-aware "
                       "objects to represent datetimes in UTC: "
-                      "datetime.datetime.fromtimestamp(t, datetime.UTC).",
+                      "datetime.datetime.fromtimestamp(t, datetime.UTC). To maintain "
+                      "compatibility datetime.datetime.fromtimestamp(timestamp) "
```

Comment:
I don't think this is right?

Reply:
If not timezone is provided, `fromtimestamp` returns a timezone unaware(?) datetime object.